### DISCUSSION TITLE
脚手架无法正常安装

### DISCUSSION THREAD

**Discussion by yeying-xingchen** [nonebot/discussions#464] [问答] [answered]
报错：--python is ignored when --force is passed. If you want to reinstall nb-cli with D:\Dev\Python\python.exe, run `pipx
reinstall nb-cli --python D:\Dev\Python\python.exe` instead.
Installing to existing venv 'nb-cli'
Fatal error from pip prevented installation. Full pip output in file:
    C:\Users\Administrator\.local\pipx\logs\cmd_2024-05-13_22.03.42_pip_errors.log

pip seemed to fail to build package:
    watchfiles~=0.16

Some possibly relevant errors from pip install:
    error: subprocess-exited-with-error
    error: metadata-generation-failed

⚠️  Not removing existing venv C:\Users\Administrator\.local\pipx\venvs\nb-cli because it was not created in this
    session
Error installing nb-cli.

**Comment by yanyongyu** [comment]
你是用的python版本是多少？是否是64位的？

**Reply by yeying-xingchen** [reply]
3.13 是64位

**Reply by yanyongyu** [reply, marked as answer]
3.13 beta 都没发布你为什么装
